perf/renaissance-jmh/build.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<project name="Renaissance suite (JMH)" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>Renaissance suite</description>
+
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/perf/renaissance-jmh" />
+	<property name="SRC" location="." />
+	
+	<target name="init">
+		<mkdir dir="${DEST}" />
+	</target>
+
+	<target name="getRenaissanceJMHSuite" depends="init">
+		<var name="FILE_NAME" value="renaissance-jmh.jar"/>
+		<if>
+		    <available file="${FILE_NAME}" type="file" />
+		    <then>
+		        <echo message="${FILE_NAME} exists. Hence, not downloading it." />
+		    </then>
+		    <else>
+				<var name="curl_options" value="-Lks -C - https://github.com/renaissance-benchmarks/renaissance/releases/download/v0.14.2/renaissance-jmh-0.14.2.jar -o ${FILE_NAME}"/>
```

Comment:
looks like tabs vs spaces in this block

Reply:
Not 100% sure on teh curl command are we checking its sig at the end?

Reply:
This build file was mostly copied from the existing Renaissance build.xml (so the problem is likely there as well). Fixing for this file. Not sure about the curl command - again, it was copied from elsewhere.

Reply:
Yeah let's leave it as it is, ultimately it's done this way upstream but we could improve it there at a later date.